onvm/onvm_nflib/onvm_event.h

```diff
@@ -135,8 +144,12 @@ int nf_subscribed_to_event(struct event_tree_node *root, uint64_t event_id, uint
 struct event_tree_node *get_event(struct event_tree_node *root, uint64_t event_id);
 
 void test_sending_event(uint64_t event_id, uint16_t dest_id);
+void sending_event(uint64_t event_id, uint16_t dest_id, struct rte_mbuf *pkt);
+void sending_event_data(uint64_t event_id, uint16_t dest_id, void *pkt);
```

Comment:
I don't understand the difference between these. Do we need both?  It seems like the second one is more general.

Don't use terms like `sending...` in your function names. In our coding style we use present tense words, so these would just be `send_event` or `send_event_data`.